int8 left_right # 'a' -> -1 = left, 'd' -> 1 = right, 0 = noop
int8 up_down # 's' -> -1 = down, 'w' -> 1 = up, 0 = noop